tests/test_create.py

```diff
@@ -1,17 +1,21 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import, division, print_function
 
-import pytest
 from contextlib import contextmanager
 from glob import glob
 from logging import getLogger, Handler
 from os.path import exists, isdir, join, relpath
 from shlex import split
 from shutil import rmtree
+import subprocess
+import sys
 from tempfile import gettempdir
 from unittest import TestCase
 from uuid import uuid4
 
+from menuinst.win32 import dirs as win_locations
```

Comment:
Oh, the import is in test code.  Yeah, in-line it, and then use pytest skipif.

Generally, in-line imports should be used only in very special cases in non-test code.  In test code, use all the in-line imports you want.